ingestion/src/test/java/feast/ingestion/service/CompositeSpecService.java

```diff
@@ -0,0 +1,78 @@
+package feast.ingestion.service;
+
+import com.google.common.collect.Lists;
+import feast.specs.EntitySpecProto.EntitySpec;
+import feast.specs.FeatureSpecProto.FeatureSpec;
+import feast.specs.StorageSpecProto.StorageSpec;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CompositeSpecService implements SpecService {
```

Comment:
Why is the `CompositeSpecService` needed for testing? Is it not sufficient to use only `MockSpecService` or `FileSpecService`?

Reply:
Oh yeah. That's because I need to pass in a store spec that isn't in a file during the Redis tests.

Previously I'd inject them afterwards, but Specs it tries to load referenced storage specs (from features) on construction now and aborts if it can't.

Reply:
Is there a reason it is not possible for that spec to be in a file? Seems a bit extra